src/utils/withAltContext.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 export default function withAltContext(flux) {
-  return function (Component) {
+  return function(Component) {
```

Comment:
We can do whichever.  Right now we have some instances with the space and some without. So I can add the rule so that we require the space so that we can be consistent everywhere.